src/main/scala/uk/gov/nationalarchives/downloadfiles/FileUtils.scala

```diff
@@ -6,7 +6,10 @@ import java.util.UUID
 
 import com.amazonaws.services.lambda.runtime.events.models.s3.S3EventNotification.S3EventNotificationRecord
 import com.typesafe.config.{Config, ConfigFactory}
```

Comment:
Think this is an un-used import: `import com.typesafe.config.{Config, ConfigFactory}`

Reply:
Removed. Thanks.